src/helpers/moveToIndex.ts

```diff
@@ -0,0 +1,15 @@
+export const moveToIndex = <T>(
+  items: T[],
+  itemToMoveIndex: number,
+  destinationIndex: number
+): T[] => {
+  if (!items.length) return items;
+
+  if (itemToMoveIndex > items.length || destinationIndex > items.length)
+    throw new Error("index should be smaller then array length");
```

Comment:
can you just `return items` please, it's in the philosophy of the package to not throw errors, you never know the UI could be outdated and the user would get an error screen instead of a no-op

Reply:
Okay, definitely! Let me adjust that